packages/utilities/src/collections/index.js

```diff
@@ -7,7 +7,9 @@
 // to be re-used by multiple rules.
 
 // A collection of locations of top-level response schemas
-const responseSchemas = ['$.paths[*][*][responses][*].content[*].schema'];
+const responseSchemas = [
+  '$.paths[*][get,put,post,delete,options,head,patch,trace].[responses][*].content[*].schema',
+];
```

Comment:
Looks good but if all collections should follow this pattern, we'd need to do the same for `schemas`, `unresolvedResponseSchemas`, `unresolvedRequestBodySchemas`, and `unresolvedSchemas`.

We may be able to reuse the "operations" collection somehow. I don't know if there are certain collections that need to operate only on specific operations but if not, I think we could do something like this to simplify:

```
const responseSchemas = [
  `${operations[0]}.[responses][*].content[*].schema`,
];
```

Reply:
done